### ISSUE TITLE
Request undefined

### ISSUE THREAD

**Issue by piliugin-anton** [kartikk221/hyper-express#101]
I saw you found a tricky way to instantiate classes more performant. Also you have a typo in Response.js, line 1001, there should be Response.prototype, not Request.prototype

**Comment by BobbyWibowo** [comment]
Just got this with v6.4.3 as well
Relevant stacktrace
```
Uncaught Exception: ReferenceError: Request is not defined
    at method (/home/bobby/src/safe.fiery.me/node_modules/hyper-express/src/components/http/Response.js:1001:13)
    at /home/bobby/src/safe.fiery.me/node_modules/hyper-express/src/shared/operators.js:117:31
    at Array.forEach (<anonymous>)
    at inherit_prototype (/home/bobby/src/safe.fiery.me/node_modules/hyper-express/src/shared/operators.js:107:30)
    at /home/bobby/src/safe.fiery.me/node_modules/hyper-express/src/shared/operators.js:103:57
    at Array.forEach (<anonymous>)
    at inherit_prototype (/home/bobby/src/safe.fiery.me/node_modules/hyper-express/src/shared/operators.js:103:42)
    at Object.<anonymous> (/home/bobby/src/safe.fiery.me/node_modules/hyper-express/src/components/http/Response.js:977:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
```

**Comment by kartikk221** [comment]
Good catch, for some reason, I was able to run the tests fine on Node v18 on my end hence this update even went out with a typo in the first place. Regardless, I've gone ahead and fixed the typo/pushed other changes in v6.4.4. Let me know If the issue still persists.
